src/lib/y2storage/encryption_processes/pervasive.rb

```diff
@@ -72,12 +97,13 @@ def pre_commit(device)
 
         master_key_file = @secure_key.filename
         sector_size = sector_size_for(device.blk_device)
+        key_size = @secure_key.secure_key_size * 8
```

Comment:
maybe here should be commadn that one size in bites and second in bytes?

Reply:
I thought it was obvious. Nevertheless, comment added.